examples/notebooks/responsibleaidashboard-diabetes-decision-making.ipynb

```diff
@@ -678,16 +677,12 @@
     "        test_data,\n",
     "        score_card_config_path\n",
     "    ):\n",
-    "        # Fetch the model\n",
-    "        fetch_job = fetch_model_component(\n",
-    "            model_id=expected_model_id\n",
-    "        )\n",
-    "        \n",
     "        # Initiate the RAIInsights\n",
     "        create_rai_job = rai_constructor_component(\n",
     "            title=\"RAI Dashboard Example\",\n",
     "            task_type=\"regression\",\n",
-    "            model_info_path=fetch_job.outputs.model_info_output_path,\n",
+    "            model_info=expected_model_id,\n",
+    "            model_input=Input(type=AssetTypes.MLFLOW_MODEL, path=azureml_model_id),\n",
```

Comment:
Is there a screenshot of what this input looks like in the UI now?  Is this a text field, or are there now models as boxes that can somehow be connected directly in the designer UI?

Reply:
shown this in meeting from last week - there's a UI bug where model shows up as data. Finding the right contacts to report bug.